### PR TITLE
Refactor map page JS code and CSS

### PR DESCRIPTION
**Pull request by MickStein**
https://eaflood.atlassian.net/browse/LTFRI-1016

Following the map changes, the JS and CSS need refactoring to be organised and condensed in order to be more maintainable.

### CODE REVIEW

**Review by JoshuaForrest — COMMENTED** [review]
Looks good!

File: node/risk-app/server/src/js/map-page/map-page.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+          event.preventDefault()
+          setCurrent(event.target.value)
+        })
+      }
+      if (measurement.name === 'map-toggle') {
+        measurement.addEventListener('change', function (event) {
+          event.preventDefault()
+          setCurrent(event.target.value)
+        })
+      }
```
Comment (Matthew-Collins-Defra) [inline node/risk-app/server/src/js/map-page/map-page.js:110]: A lot of duplication here. The event listener function could be a variable, or the multiple if statements could be a single condition
if (measurement.name === 'measurements') || (measurement.name === 'scenarios-depth') ... etc

Or you could c onstruct an array of availble names like "eventOn = ['measurements', 'scenarios-depth' ...
Then use something like
if (eventOn.indexOf(measurement.name) >= 0) { addEventListen ....

File: node/risk-app/server/src/js/map-page/map-page.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+  const easting = parseInt(getParameterByName('easting'), 10)
+  const northing = parseInt(getParameterByName('northing'), 10)
+  const hasLocation = !!easting
+
+  mapPageConsts.maps.loadMap((hasLocation && [easting, northing]))
+
+  // This function updates the map to the radio button you select (extent, depth, velocity)
+  function setCurrent (ref) {
+    const mapController = new MapController(mapCategories.categories)
```
Comment (Matthew-Collins-Defra) [inline node/risk-app/server/src/js/map-page/map-page.js:71]: If you include `window.mapCategories.categories` here, that will make explicit where the data is coming from, stop the SonarCloud warning, and remove the need for the global declaration on Line 6.

**Review by Matthew-Collins-Defra — CHANGES_REQUESTED** [review]

**Review by Matthew-Collins-Defra — APPROVED** [review]
Looks good to me! Nice tidy up.

**Pull request merged**